### PR TITLE
PL release notes for 784xx

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/harness-platform.md [hunk 1]
```diff
@@ -13,6 +13,100 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+  This update will not affect any business operations.
+
+- The [Role Assignment](https://apidocs.harness.io/tag/Role-Assignments/#operation/getFilteredRoleAssignmentByScopeList) API now includes the principal's name and email address in the response.(PL-31064, ZD-36050)
+
+- Harness now supports the integration of GCP Secrets Manager for all users. (PL-31051)
+  
+  For more information, see [Add a Google Cloud Secret Manager](https://developer.harness.io/docs/platform/Security/add-a-google-cloud-secret-manager)
+
+- There is now a limit on the number of entities that can be created for **FREE** and **COMMUNITY** users in Harness. (PL-30838)
+
```
Comment (SmritiSatya) [inline release-notes/harness-platform.md:30]: There is a limit on the number of entities that can be created for **FREE** and for **COMMUNITY** users in Harness. (PL-30838)

File: release-notes/harness-platform.md [hunk 2]
```diff
@@ -13,6 +13,100 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+  - Project - 100
+  - Secrets - 100
+  - Variables - 100
+  - User groups -100
+  - Service accounts - 100
+
+- You can now add policy sets under the **Advanced** section of each step. (PIE-7794)
+
+  In this way, you can validate step outputs generated during pipeline execution without requesting a payload explicitly. Policy sets receive steps and their outputs as payloads, with a predefined schema for every step.
+
```
Comment (SmritiSatya) [inline release-notes/harness-platform.md:42]: This way, you can validate the step outputs generated during pipeline execution without explicitly requesting a payload. Policy sets receive the steps and their outputs as payloads, with a pre-defined schema for every step.

File: release-notes/harness-platform.md [hunk 3]
```diff
@@ -13,6 +13,100 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+- Accessing the CD module redirects you to one of the following: 
+
+  - Get Started: If you don't have any pipelines in your project
+
```
Comment (SmritiSatya) [inline release-notes/harness-platform.md:46]: Get Started: If you don't have any pipelines in your project.

**Review by SmritiSatya — APPROVED** [review]

**Review by charanya-j — APPROVED** [review]

**Review by SudarshanPurohit — COMMENTED** [review]
Looks good to me.

**Pull request merged**